ons_alpha/bulletins/models.py

```diff
@@ -47,6 +46,9 @@ class BulletinPage(BasePage):
     is_accredited = models.BooleanField(default=False)
     body = StreamField(BulletinStoryBlock(), use_json_field=True)
     updates = StreamField(CorrectionsNoticesStoryBlock(), blank=True, use_json_field=True)
+    previous_version = models.ForeignKey(
```

Comment:
Issue: The previous version needs to be on the correction block (see the `updates` field). Otherwise, the "previous" version will keep getting overwritten when changes are published.

Reply:
This still needs doing.